sdk/communication/communication-phone-numbers/CHANGELOG.md

```diff
@@ -1,27 +1,21 @@
 # Release History
 
-## 1.2.0-beta.5 (Unreleased)
+## 1.2.0-beta.4 (2023-01-10)
+- Adds support for Azure Communication Services Phone Numbers Browse API Methods.
 
```

Comment:
Please add line for SIP routing:
- Adds support for Direct routing configuration management.